trunk/SUAVE/Methods/Weights/Correlations/General_Aviation/empty.py

```diff
@@ -0,0 +1,402 @@
+## @ingroup Methods-Weights-Correlations-General_Aviation
+# empty.py
+# 
+# Created:  Jan 2014, A. Wendorff
```

Comment:
Did andrew write this?

Reply:
Nope. Just missed the author strings. Updated now